packages/brick_offline_first_with_rest_build/lib/src/offline_first_rest_generators.dart

```diff
@@ -134,18 +134,23 @@ class _OfflineFirstRestDeserialize extends RestDeserialize {
             final where =
                 _convertSqliteLookupToString(offlineFirstAnnotation.where!, iterableArgument: 's');
             final getAssociationText = getAssociationMethod(argType, query: 'Query(where: $where)');
-            final getAssociations = '''($fieldValue ?? []).map((s) => $getAssociationText
-            ).whereType<Future<$argType>>()$fromRestCast''';
 
             if (checker.isArgTypeAFuture) {
-              return getAssociations;
+              return '''($fieldValue ?? []).map<Future<$argType>>((s) => $getAssociationText)$fromRestCast''';
```

Comment:
If this is one line it doesn't need to have a multiline string
```suggestion
              return '($fieldValue ?? []).map<Future<$argType>>((s) => $getAssociationText)$fromRestCast';
```